sass/animate/_easing.scss

```diff
@@ -106,18 +65,13 @@ $EASING: (
 ///   (in additional to the provided defaults)
 /// @return {length} -
 ///   The pre-defined easing
-@function ease(
-  $name: 'ease',
-  $do: null,
-  $source: $easing
-) {
-  $source: map.merge($EASING, $source);
+@function ease($name: 'ease', $do: null, $source: $easing) {
```

Comment:
When the CSSers spoke this week, I THINK we leaned towards the parameters being separated by a line break so it is easier to scan/see which changes were made, etc. I used to be a "everything all on line line" kind of person but have since found line breaks helpful in certain situations. 

What do you think about this moving forward @dvdherron @mirisuzanne? Is there a preference that we decided upon?

If so, is there a setting that we need @oluoluoxenfree  @jgerigmeyer for that to take place?

Reply:
@stacyk If we want to use Prettier to auto-format, I don't think there's an option for this. If we want to ditch Prettier and only use Stylelint, then we have more control but will need to do more of the formatting ourselves.

Reply:
I like having parameters on their own lines, with trailing commas -- but I'm not sure if there's a setting for it? Seems like there must be somewhere, since David's local linter was changing this the other way…

Reply:
Yes, there are stylelint rules for it, but only if we remove Prettier. That's your call.

https://github.com/prettier/stylelint-prettier